CHANGELOG.md

```diff
@@ -96,9 +114,10 @@ Please add a _short_ line describing the PR you make, if the PR implements a spe
 * Add a `--token-path` argument to tell where the token should be saved and which token to be used. ([#329](https://github.com/ScilifelabDataCentre/dds_cli/pull/329))
 * Remove `--username` option ([#331](https://github.com/ScilifelabDataCentre/dds_cli/pull/331))
 * Add support for the zero-conf environment in dds_web ([#337](https://github.com/ScilifelabDataCentre/dds_cli/pull/337))
-* Increase request timeout to 30 ([#344]https://github.com/ScilifelabDataCentre/dds_cli/pull/344)
+* Increase request timeout to 30 ([#344]<https://github.com/ScilifelabDataCentre/dds_cli/pull/344>)
```

Comment:
Did you mean to put `<>` and not `()`?

Reply:
```suggestion
* Increase request timeout to 30 ([#344](https://github.com/ScilifelabDataCentre/dds_cli/pull/344))
```

Reply:
Nope, my editor automatically fixes markdown so I bet that I hit save before I finished writing the markdown and it surrounded the URL for me..

Reply:
(naked URLs should have < url > for properly valid markdown)

Reply:
Ah it's not my changelog - ok then yeah it was broken markdown before (no brackets) and it automatically fixed it in a dumb way.. 😆